apps/api/src/schema/grades.ts

```diff
@@ -2,7 +2,7 @@ import { z } from "@hono/zod-openapi";
 import { courseLevels, terms } from "@packages/db/schema";
 import { yearSchema } from "./lib";
 
-const geCategories = [
+export const geCategories = [
```

Comment:
maybe consider moving this to a common file and have ap-exams and this depend on that new file

Reply:
done, still a lot of duplication of ge categories and other things so that'll be another issue